includes/admin_import.php

```diff
@@ -4,743 +4,1212 @@
+								<div class="progress-bar"></div>
+							</div>
+							<ol id="tsml_import_errors" class="error inline hidden"></ol>
+						<?php } ?>
+
+						<?php if (empty($tsml_mapbox_key) && empty($tsml_google_maps_key)) { ?>
+							<div class="notice notice-warning inline" style="margin: 1px 0 22px;">
+								<div class="inside" >
+									<h1>Enable Maps on Your Site</h1>
+									<p>If you want to enable maps on your site you have two options: <strong>Mapbox</strong> or <strong>Google</strong>. *See the Google restriction noted below.
+										They are both good options, although Google is not completely supported by all our features! In all likelihood neither one will charge you money. Mapbox gives
+										<a href="https://www.mapbox.com/pricing/" target="_blank">50,000 free map views</a> / month, Google gives
+										<a href="https://cloud.google.com/maps-platform/pricing/" target="_blank">28,500 free views</a>.
+										That's a lot of traffic!
+									</p>
+									<p>To sign up for Mapbox <a href="https://www.mapbox.com/signup/" target="_blank">go here</a>. You will only need
+										a valid email address, no credit card required. Copy your access token and paste it below:</p>
+									<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px 0;">
+										<?php wp_nonce_field($tsml_nonce, 'tsml_nonce', false) ?>
+										<div class="input">
+											<input type="text" name="tsml_add_mapbox_key" placeholder="Enter Mapbox access token here">
+										</div>
+										<div class="btn">
+											<input type="submit" class="button" value="<?php _e('Add', '12-step-meeting-list') ?>">
+										</div>
+									</form>
+
+									<p>*Please note: Only Mapbox is supported by our <b>TSML UI</b> user interface!  Google is not an option.
+
+									<p>For our legacy user interface (<b>Legacy UI</b>), you can alternatively choose to use Google. Their interface is slightly more complex because they offer more
+										services. <a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">Go here</a>
+										to get a key from Google. The process should only take a few minutes, although you will have to enter a	credit card.
+										<a href="https://theeventscalendar.com/knowledgebase/setting-up-your-google-maps-api-key/" target="_blank">Here
+											are some instructions</a>.
+									</p>
+
+									<p>Be sure to:<br>
+										<span class="dashicons dashicons-yes"></span> Enable the Google Maps Javascript API<br>
+										<span class="dashicons dashicons-yes"></span> Secure your credentials by adding your website URL to the list
+										of allowed referrers
+									</p>
+
+									<p>Once you're done, paste your new key below.</p>
+
+									<form class="columns" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>" style="margin:15px -5px 22px 0;">
```

Comment:
would be good to keep all CSS in CSS files

Reply:
Still TODO